docs/csharp/language-reference/operators/conditional-operator.md

```diff
@@ -50,19 +56,19 @@ The following example demonstrates the usage of the conditional operator:
 
 ## Conditional ref expression
 
-Beginning with C# 7.2, a [ref local](../keywords/ref.md#ref-locals) or [ref readonly local](../keywords/ref.md#ref-readonly-locals) variable can be assigned conditionally with the conditional ref expression. You can also use the conditional ref expression as a [reference return value](../keywords/ref.md#reference-return-values) or as a [`ref` method argument](../keywords/ref.md#passing-an-argument-by-reference).
+Beginning with C# 7.2, a [ref local](../keywords/ref.md#ref-locals) or [ref readonly local](../keywords/ref.md#ref-readonly-locals) variable can be assigned conditionally with a conditional ref expression. You can also use a conditional ref expression as a [reference return value](../keywords/ref.md#reference-return-values) or as a [`ref` method argument](../keywords/ref.md#passing-an-argument-by-reference).
 
-The syntax for the conditional ref expression is as follows:
+The syntax for a conditional ref expression is as follows:
 
 ```csharp
 condition ? ref consequent : ref alternative
 ```
 
-Like the original conditional operator, the conditional ref expression evaluates only one of the two expressions: either `consequent` or `alternative`.
+Like the original conditional operator, a conditional ref expression evaluates only one of the two expressions: either `consequent` or `alternative`.
 
-In the case of the conditional ref expression, the type of `consequent` and `alternative` must be the same.
+In the case of a conditional ref expression, the type of `consequent` and `alternative` must be the same. Conditional ref expressions are not target-typed.
```

Comment:
@BillWagner please confirm that addition.

Reply:
That is correct. `ref` expressions must be the same.